packages/squidex/schema/crn/schemas/research-outputs.json

```diff
@@ -242,6 +242,26 @@
           "isHalfWidth": false
         }
       },
+      {
+        "name": "reviewRequestedBy",
+        "isHidden": false,
+        "isLocked": false,
+        "isDisabled": false,
```

Comment:
I believe it would be better for this to be disabled as well  (like updatedBy)  :)

Reply:
Updated